sdks/python/apache_beam/ml/inference/sklearn_inference.py

```diff
@@ -41,7 +41,9 @@ class ModelFileType(enum.Enum):
   JOBLIB = 2
 
 
-class SklearnInferenceRunner(InferenceRunner):
+class SklearnInferenceRunner(InferenceRunner[numpy.ndarray,
+                                             PredictionResult,
+                                             Any]):
```

Comment:
Correct me if I'm wrong @ryanthompson591, but I think the base class for sklearn model is `BaseEstimator` (from sklearn.base import BaseEstimator). Could we use this for the type instead of `Any`?

Reply:
Cool. Just double checked that `BaseEstimator` [1] is indeed the base class for all estimators in scikit-learn so I think we can update it

[1] https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html

Reply:
yes, BaseEstimator looks right. I also checked that pipelines inherit this base class.